TLM/TLM/Custom/PathFinding/CustomPathFind.cs

```diff
@@ -3052,6 +3120,12 @@ private struct BufferItem {
                             }
                         }
 
+                        if ((nextLaneInfo.vehicleCategory & VehicleInfo.VehicleCategory.PublicTransportRoad) != 0 &&
+                            (nextLaneInfo.vehicleCategory & ~(VehicleInfo.VehicleCategory.Bus | VehicleInfo.VehicleCategory.Trolleybus | VehicleInfo.VehicleCategory.Taxi)) == 0)
+                        {
+                            nextItem.ComparisonValue /= 100f;
```

Comment:
if this is critical path I prefer `*= 0.01f`

Reply:
that's vanilla code, actually